examples/ex5/ex5.cpp

```diff
@@ -190,6 +191,8 @@ int main(int argc, char **argv) {
       printf("------Plot %d: WriteVTU by rank : %d\n", plot_counter,
              world_rank);
       WriteVTU(argv[1], plot_counter, Time);
+      stepTime[nStep] = Time;
+      WritePVD(argv[1], nStep, stepTime);
```

Comment:
Is a call to WritPVD required here?

Reply:
We want to write to PVD after each plot, not only in the end. If there is somewhere else we might prefer writing PVD, could you let me know?